### ISSUE TITLE
Exaile accessibility issues with screen readers

### ISSUE THREAD

**Issue by inspector3535** [exaile/exaile#968]
Greetings,
I just discovered the Exaile app via the Scoop package manager, but I'm encountering accessibility issues when using it with the NVDA screen reader. I am unable to navigate the interface or access any of the application's features.
I am using NVDA version 2024.4.2 (2024.4.2.35031) with Exaile version 4.1.4.
OS: Windows 10 22H2 (Build 19045.5854)
The following log information from NVDA indicates potential issues:
```Developer info for navigator object:
name: 'Exaile'
role: UNKNOWN
processID: 420
roleText: None
states: FOCUSABLE, FOCUSED
isFocusable: True
hasFocus: True
Python object: <NVDAObjects.Dynamic_ClipboardCommandAnnouncementContentGenericClientIAccessibleWindowNVDAObject object at 0x05684130>
Python class mro: (<class 'NVDAObjects.Dynamic_ClipboardCommandAnnouncementContentGenericClientIAccessibleWindowNVDAObject'>, <class 'globalPlugins.NVDAExtensionGlobalPlugin.clipboardCommandAnnouncement.ClipboardCommandAnnouncement'>, <class 'NVDAObjects.IAccessible.ContentGenericClient'>, <class 'NVDAObjects.IAccessible.IAccessible'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'documentBase.TextContainerObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <class 'garbageHandler.TrackedObject'>, <class 'object'>)
description: None
location: RectLTWH(left=0, top=23, width=1366, height=705)
value: ''
TextInfo: <class 'displayModel.DisplayModelTextInfo'>
appModule: AppModule(appModuleHandler, appName='exaile', processID=420)
appModule.productName: exception: No version information
appModule.productVersion: exception: No version information
appModule.helperLocalBindingHandle: c_long(106840320)
appModule.appArchitecture: 'x86'
windowHandle: 788106
windowClassName: 'gdkWindowToplevel'
windowControlID: 0
windowStyle: 399441920
extendedWindowStyle: 272
windowThreadID: 2280
windowText: 'Exaile'
displayText: ''
IAccessibleObject: <POINTER(IAccessible) ptr=0x662fac0 at 5663990>
IAccessibleChildID: 0
IAccessible event parameters: windowHandle=788106, objectID=-4, childID=0
IAccessible accName: 'Exaile'
IAccessible accRole: ROLE_SYSTEM_CLIENT
IAccessible accState: STATE_SYSTEM_FOCUSED, STATE_SYSTEM_FOCUSABLE, STATE_SYSTEM_VALID (1048580)
IAccessible accDescription: None
IAccessible accValue: None```
 
The log indicates that most UI elements are defined as ROLE_SYSTEM_CLIENT, which makes it difficult for screen readers to identify and interpret them. Also, the role of the navigator object is reported as UNKNOWN. The errors appModule.productName: exception: No version information and appModule.productVersion: exception: No version information suggest that NVDA is unable to retrieve the application's version information.
I would appreciate it if you could consider adding proper accessibility support to Exaile. I am willing to help with testing and providing feedback to ensure the application is accessible to screen reader users.
Best regards.
